apps/condo/domains/organization/utils/serverSchema/Organization.js

```diff
@@ -93,31 +96,52 @@ async function findOrganizationEmployee (context, query) {
 
 const salesCRMRequestLogger = pino({ name: 'sales_crm', enabled: falsey(process.env.DISABLE_LOGGING) })
 
-async function pushToSalesCRM (organization) {
-    if (!SALES_CRM_WEBHOOK_URL) {
-        return salesCRMRequestLogger.warn('SALES_CRM_WEBHOOK_URL not specified correctly in config')
+async function pushOrganizationToSalesCRM (organization) {
+    if (!SALES_CRM_ORGANIZATIONS_WEBHOOK_URL) {
+        return salesCRMRequestLogger.warn('SALES_CRM_ORGANIZATIONS_WEBHOOK_URL not specified correctly in config')
```

Comment:
You don’t need to log it. It’s annoying in development

Reply:
Done